project/Doc.scala

```diff
@@ -211,7 +211,7 @@ object Unidoc extends AutoPlugin {
   // down to two assuming https://github.com/typesafehub/genjavadoc/issues/66 is possible.
   override lazy val projectSettings = inConfig(Genjavadoc)(Defaults.configSettings) ++ Seq(
     ivyConfigurations += GenjavadocCompilerPlugin,
-    libraryDependencies += ("com.typesafe.genjavadoc" % "genjavadoc-plugin" % "0.15" % "genjavadocplugin->default(compile)")
+    libraryDependencies += ("com.typesafe.genjavadoc" % "genjavadoc-plugin" % "0.16" % "genjavadocplugin->default(compile)")
```

Comment:
today I learn that genjavadoc-plugin uses the full scala version. Updated to 2.12.13 breaks because there is not 0.15_2.12.13.